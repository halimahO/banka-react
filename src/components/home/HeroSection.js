import React from 'react';

const HeroSection = () => {
  return (
    <div className="section-hero">
      <div className="container-hero">
        <h1 className="bg-text">Banka</h1>
        <h2 className="md-text">
          A banking app that guarantees the best banking experience.
        </h2>
        <div className="hero-button">
          <div className="hero-btn inline-style">
            <a href="register.html">Get started</a>
          </div>
        </div>
      </div>
    </div>
  );
};
export default HeroSection;
