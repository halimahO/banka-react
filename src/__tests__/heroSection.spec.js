import React from 'react';
import { shallow } from 'enzyme';
import HeroSection from '../components/home/HeroSection';

const wrapper = shallow(<HeroSection />);

describe('<HeroSection/> rendering', () => {
  it('should render HeroSection without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
