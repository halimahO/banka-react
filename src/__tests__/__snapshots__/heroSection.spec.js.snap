// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeroSection/> rendering should render HeroSection without crashing 1`] = `ShallowWrapper {}`;
